services/frontend-service/src/ui/components/ServiceLane/ServiceLane.test.tsx

```diff
@@ -94,9 +94,56 @@ describe('Service Lane', () => {
 });
 
 const data = [
+    {
+        name: 'test same version',
+        diff: '0',
+        releases: [
+            {
+                version: 1,
+                sourceMessage: 'test1',
+                sourceAuhor: 'test',
+                sourceCommitId: 'commit1',
+                createdAt: new Date(2002),
+            },
+        ] as any,
```

Comment:
why `as any` and not `as Release[]`?

Reply:
I copied it like this from another code, I'll fix t now